core/src/main/java/feign/RequestTemplate.java

```diff
@@ -291,6 +308,33 @@ public RequestTemplate append(CharSequence value) {
     return this;
   }
 
+  /**
+   * Append the specified {@code value} to {@code url} without decoding or encoding. Append all 
```

Comment:
lots of doc. but no example. forward referencing to ribbon probably won't help people understand.

Try to shorten this to maybe two succinct sentences and an example.

Ex. 

```
Like {@link #foo} except ..

<p> <-- note p not br and the newline break makes it easy to read, too..
ex if your input is X unless you call this method, it would become Y
```